tensorflow/lite/micro/micro_interpreter.cc

```diff
@@ -326,6 +397,117 @@ TfLiteStatus MicroInterpreter::RequestScratchBufferInArena(TfLiteContext* ctx,
       bytes, interpreter->graph_.GetCurrentSubgraphIndex(), buffer_idx);
 }
 
+TfLiteStatus MicroInterpreter::ResizeTensor(TfLiteContext* context,
```

Comment:
One of the design decisions we made with TFL Micro was only to support graphs with fixed shapes throughout. Can you add some context about why this is needed in this case?

Reply:
lstm prepare function uses ResizeTensor(), so we implemented this and it is same as TFLite implementation